embeddings/data/datamodule.py

```diff
@@ -103,13 +103,12 @@ def prepare_data(self) -> None:
 
     def setup(self, stage: Optional[str] = None) -> None:
         self.dataset = self.load_dataset()
-        self.downsample_dataset()
```

Comment:
maybe not now but consider comments and logs to communicate to users what's happening

Reply:
It's general setup function which is in `pytorch_lightning` package. Here we just load dataset and we converts it to torch format. Although we can add logging here, what is your opinion?

Reply:
Yes, it's standard function but nonetheless that's good to let users know at what processing is -thanks to logging libraries these can be set to output in verbose mode only for example